src/index.tsx

```diff
@@ -17,10 +17,10 @@ import {
   PanGestureHandlerEventExtra
 } from "react-native-gesture-handler";
 import Animated from "react-native-reanimated";
-import { getOnCellTap, springFill, setupCell } from "./procs";
+import { springFill, setupCell } from "./procs";
 import listenToKeyboardEvents from "./KeyboardAware";
 
-const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
+const AnimatedFlatList = Animated.createAnimatedComponent(RNFlatList);
```

Comment:
The Reanimated Flatlist seemed to have issues calculating component heights when multiple lists were nested, but that wasn't the case for the stock RN version.